specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2025-11-01-preview/cosmos-db.json

```diff
@@ -8612,6 +8612,16 @@
         "enableAllVersionsAndDeletesChangeFeed": {
           "description": "Flag to indicate if All Versions and Deletes Change feed feature is enabled on the account",
           "type": "boolean"
+        },
+        "throughputPoolDedicatedRUs": {
+          "type": "long",
+          "format": "int64",
+          "description": "Dedicated amount of RUs for database account."
```

Comment:
Basic question - do we need to say anywhere this is a read-only property, so customers don't think they can edit this property and change the behavior of the RU/s in the resources of the account?